modules/oe_bootstrap_theme_helper/src/TwigExtension/TwigExtension.php

```diff
@@ -235,4 +251,33 @@ public function toInternalLanguageId(string $language_code): string {
     return $language_code;
   }
 
+  /**
+   * Alter a link with BCL logic.
+   *
+   * @param \Twig\Environment $env
+   *   The env.
+   * @param string $label
+   *   The link text for the anchor tag as a translated string.
+   * @param \Drupal\Core\Url|string $path
+   *   The URL object or string used for the link.
+   * @param array|\Drupal\Core\Template\Attribute $attributes
+   *   An optional array or Attribute object of link attributes.
+   *
+   * @return array
+   *   The link render array.
+   */
+  public function bclLink(Environment $env, $label, $path, Attribute $attributes): array {
+    if (empty($path)) {
+      $path = 'route:<nolink>';
+    }
+
+    // Here we handle internal urls which already had toString() called.
+    if (is_string($path) && !UrlHelper::isExternal($path)) {
+      $path = str_replace(base_path(), '/', $path);
```

Comment:
didn't find anything in core, closes it something in FileUrlGenerator but its not the same.